### ISSUE TITLE
After setup SMTP relay; errors "mail: invalid option"

### ISSUE THREAD

**Issue by lantrix** [nextcloud/vm#1687]
## Steps to reproduce
1. Download the nextcloud VMware / VirtualBox (.OVA) Nextcloud VM Pre-installed – 40 GB **v20.0.1**
2. Run through a standard install, using an SMTP mail relay
3. Initial test works to SMTP relay
4. Every subsequent "Sending" on the console shows an error

## Expected behaviour

Mail sent after the test email received is also received

## Actual behaviour

No mail sent during setup is received, except initial test.
Every time the setup says it is sending mail there is an error:

```
mail: invalid option -- '-'
```

## Server configuration

**Nextcloud server version:** Nextcloud 20.0.2 (although installer was 20.0.1 and it updated)

**Server OS** Ubuntu server

**How did you install the VM?** Released version

## Network
**Do you use DHCP?** Yes

**Is port 80 and/or 443 open?** Yes

## Logs / Screenshots

Received test
<img width="810" alt="nc-mail-test-worked" src="https://user-images.githubusercontent.com/117008/99623280-dd7b7180-2a7f-11eb-8357-3ef71aefa07b.png">

On of the many exact same errors during setup sending mail
![nc-send-error](https://user-images.githubusercontent.com/117008/99623516-685c6c00-2a80-11eb-8e98-6bb609ae665f.png)

#### Log file (/var/log/nextcloud/nextcloud.log)

https://0bin.net/paste/hvOTicSf#dqg4206hr85qg7tXoC7o32vqpP1151EoX7V+I6rgr4p

#### Installation log (command output)

N/A here

**Comment by enoch85** [comment]
cc @szaimen

**Comment by szaimen** [comment]
@lantrix I can unfortunately not reproduce this. Have you tried to reinstall it? Maybe this fixes the problem...

Could you please post your mail logs:
`/var/log/msmtp`

Also it would be interesting what is inside 
`/etc/mail.rc`

Thanks!

**Comment by lantrix** [comment]
Thanks @szaimen I'll reinstall it, and see if it happens again.
I'll also grab a copy of the logs if it does again.

**Comment by lantrix** [comment]
Did the reinstall of 20.0.2 and same problem exactly. Into email works, rest error with:
```
mail: invalid option -- '-'
usage: mail [-dEIinv] [-a header] [-b bcc-addr] [-c cc-addr] [-r from-addr] [-s subject] [--] to-addr ...
       mail [-dEIiNnv] -f [name]
       mail [-dEIiNnv] [-u user]
```

`/var/log/msmtp` is:

```
mail: /usr/bin/msmtp -t: No such file or directory
Can't send mail: sendmail process failed with error code 1
Dec 06 22:49:15 host=smtp.pobox.com tls=on auth=on user=my@email.com from=my@email.com recipients=my@email.com mailsize=1119 smtpstatus=250 smtpmsg='250 2.0.0 Ok: queued as 6C1E89EEA3' exitcode=EX_OK
```

`/etc/mail.rc` is:

```
set sendmail="/usr/bin/msmtp"
```

**Comment by lantrix** [comment]
I've manually imported the `lib.sh` into a root session to test `send_mail "subject" "message"` [function in the library](https://github.com/nextcloud/vm/blob/master/lib.sh#L1525-L1541).

Same error

```
root@cloud:~# send_mail "subject" "text"
Sending 'subject' to my@email.com
mail: invalid option -- '-'
usage: mail [-dEIinv] [-a header] [-b bcc-addr] [-c cc-addr] [-r from-addr] [-s subject] [--] to-addr ...
       mail [-dEIiNnv] -f [name]
       mail [-dEIiNnv] [-u user]
```

Breaking down the function and running it manually, [this line is the one](https://github.com/nextcloud/vm/blob/master/lib.sh#L1534) failing:

```
root@cloud:~# RECIPIENT=$(grep "recipient=" /etc/msmtprc)
root@cloud:~# RECIPIENT="${RECIPIENT##*recipient=}"
root@cloud:~# echo -e "test message" | mail --subject "NcVM - test subject" "$RECIPIENT"
mail: invalid option -- '-'
usage: mail [-dEIinv] [-a header] [-b bcc-addr] [-c cc-addr] [-r from-addr] [-s subject] [--] to-addr ...
       mail [-dEIiNnv] -f [name]
       mail [-dEIiNnv] [-u user]
```

This DOES work:

```
echo -e "test message" | mail -s "NcVM - test subject" "$RECIPIENT"
```

**Comment by lantrix** [comment]
The reason the welcome email works is it already uses `-s` [in the script](https://github.com/nextcloud/vm/blob/master/addons/smtp-mail.sh#L235) `smtp-mail.sh`

**Comment by szaimen** [comment]
Fixed in #1710
